packages/11ty/_includes/components/navigation.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(eleventyConfig) {
     const isHomePage = currentPage.url === home
 
     // @TODO figure out js module-friendly filters -- this one should work though
-    const truncate = (text, limit) => text.slice(0, limit)
+    const truncate = (text, limit) => text?.slice(0, limit)
```

Comment:
Same as previous comment re: `page.data.content` being undefined for a simple word count -- this was throwing errors on build so I made it optional, but this should be actually implemented and not just optionally skipped so the build works 🙃